doc/source/whatsnew/v0.18.1.txt

```diff
@@ -48,6 +50,48 @@ see :ref:`Custom Business Hour <timeseries.custombusinesshour>` (:issue:`11514`)
 Enhancements
 ~~~~~~~~~~~~
 
+.. _whatsnew_0181.deferred_ops:
+
+``.groupby(..)`` syntax with window and resample operations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``.groupby(...)`` has been enhanced to provide convenient syntax when working with ``.rolling(..)``, ``.expanding(..)`` and ``.resample(..)`` per group, see (:issue:`12486`, :issue:`12738`).
+
+.. ipython:: python
+
+   df = pd.DataFrame({'A': [1] * 20 + [2] * 12 + [3] * 8,
+                      'B': np.arange(40)})
+   df
+
+You can now use ``.rolling(..)`` and ``.expanding(..)`` as methods on groupbys. These return another object where you operate.
+
+Previously you would have to do this to get a rolling window mean per-group:
+
+.. ipython:: python
+
+   df.groupby('A').apply(lambda x: x.rolling(4).B.mean())
+
+Now you can do:
+
+.. ipython:: python
+
+   df.groupby('A').rolling(4).B.mean()
+
+For ``.resample(..)`` type of operations, previously you would have to:
```

Comment:
it worked partially, but only for downsampling. I'll give a different example which illustrates.